packages/editor/src/components/post-card-panel/index.js

```diff
@@ -36,13 +36,13 @@ export default function PostCardPanel( { className, actions, children } ) {
 		const { getEditedEntityRecord } = select( coreStore );
 		const _type = getCurrentPostType();
 		const _id = getCurrentPostId();
-		let _templateInfo;
 		const _record = getEditedEntityRecord( 'postType', _type, _id );
+		const _templateInfo = __experimentalGetTemplateInfo( _record );
 		return {
 			title: _templateInfo?.title || getEditedPostAttribute( 'title' ),
 			modified: getEditedPostAttribute( 'modified' ),
 			id: _id,
-			templateInfo: __experimentalGetTemplateInfo( _record ),
+			templateInfo: _templateInfo,
```

Comment:
Here the original code looked wrong. Create a `_templateInfo` variable that is never assigned to, access the `_templateInfo?.title` field that is never there...

Reply:
Good catch!